default.nix

```diff
@@ -1,12 +1,77 @@
 with (import ./nix/nixpkgs.nix) {};
 
-buildGoPackage rec {
-    name = "dividat-driver";
-    goPackagePath = "dividat-driver";
+let
 
-    src = ./src/dividat-driver;
+  version = "2.2.0-dev";
+  channel = "develop";
```

Comment:
This seems odd to me. The channel is where it's released, not a function of the code.

What would be the flow working with this? I want to release a version, update the `master` branch from `develop`, then change `channel` to `"master"` and release, then merge the changers from `master` back into `develop`, where I change `channel` back to `"develop"`?

Reply:
I think channel is part of code as driver executable is locked to a specific channel (updates).

> What would be the flow working with this? I want to release a version, update the master branch from develop, then change channel to "master" and release, then merge the changers from master back into develop, where I change channel back to "develop"?

I use `git merge --no-commt`. That is merge, update version/channel and then commit.

Reply:
Let's make the `channel` and possibly `version` parameters to `nix-build`. Either use obviously non-release fallback values or fail if parameters are not set.